token/services/certifier/interactive/client.go

```diff
@@ -138,25 +138,17 @@ func (d *CertificationClient) Scan() {
 			logger.Debugf("request certification of [%v] satisfied with no error", toBeCertified)
 		}
 
-		done := false
-		for {
+		for done := false; !done; {
```

Comment:
mhhhhh .... while this maybe fixes the break .... the entire methods should be improved to get rid of the "for in for" situation.

Reply:
can we move this for to another accessory function?

Reply:
Sure I also try to avoid nested for's.